homeassistant/components/tessie/strings.json

```diff
@@ -0,0 +1,102 @@
+{
+  "config": {
+    "error": {
+      "invalid_access_token": "[%key:common::config_flow::error::invalid_access_token%]",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    },
+    "step": {
+      "user": {
+        "data": {
+          "access_token": "[%key:common::config_flow::data::access_token%]"
+        },
+        "description": "Enter your access token from [my.tessie.com/settings/api](https://my.tessie.com/settings/api)."
+      },
+      "reauth_confirm": {
+        "data": {
+          "access_token": "[%key:common::config_flow::data::access_token%]"
+        },
+        "description": "Enter your access token from [my.tessie.com/settings/api](https://my.tessie.com/settings/api)",
```

Comment:
reauth leftovers

Also you can save the translators some time and reference the description in the next PR like https://github.com/home-assistant/core/blob/6a6956573fd534feae497b3773401faddbca7860/homeassistant/components/zwave_js/strings.json#L78

Reply:
Oh yeah sure thing, I also removed "already_configured" since it doesnt do that either.